streaming.go

```diff
@@ -89,6 +89,7 @@ func (c *Client) streaming(ctx context.Context, p string, params url.Values) (ch
 			}
 			req, err := http.NewRequest(http.MethodGet, u.String(), in)
 			if err == nil {
+				req = req.WithContext(ctx)
```

Comment:
yes, required. there are two chance to handle cancel in this part. the first is while sending request. second is while reading. the second already implemented but first is not. so please remain your code.